acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/DataDirectoryFeatureFlagAcceptanceTest.java

```diff
@@ -89,7 +90,7 @@ public static void tearDownBase() {
   }
 
   @Test
-  public void valueTransfer() {
+  public void valueTransfer() throws IOException {
```

Comment:
Why add `IOException`? (this PR has added no lines to introduce the exception)

Reply:
ups, that happened during the merge ... Will check where the exceptions were necessary!!
This one is needed!

Reply:
After fixing the Accounts and Transactions classes this is no longer needed!